instrumentation/opentelemetry-instrumentation-httpx/src/opentelemetry/instrumentation/httpx/__init__.py

```diff
@@ -315,12 +324,16 @@ def handle_request(
         httpx.Response,
     ]:
         """Add request info to span."""
-        if context.get_value("suppress_instrumentation"):
-            return self._transport.handle_request(*args, **kwargs)
-
         method, url, headers, stream, extensions = _extract_parameters(
             args, kwargs
         )
+
+        if self._excluded_urls and self._excluded_urls.url_disabled(url):
+            return self._transport.handle_request(*args, **kwargs)
+
+        if context.get_value("suppress_instrumentation"):
+            return self._transport.handle_request(*args, **kwargs)
+
```

Comment:
You could move that part were it was before. If instrumentation is suppressed then we don't need to bother with anything else, starting from `_extract_parameters`